pyt/__main__.py

```diff
@@ -30,6 +30,24 @@
 )
 
 
+def discover_files(targets, excluded_files, recursive=False):
+    included_files = list()
+    excluded_list = excluded_files.split(",")
+
+    for target in targets:
+        if os.path.isdir(target):
+            if recursive:
+                for root, dirs, files in os.walk(target):
+                    for f in files:
+                        fullpath = os.path.join(root, f)
+                        if os.path.splitext(fullpath)[1] == '.py' and fullpath.split("/")[-1] not in excluded_list:
+                            included_files.append(fullpath)
+        else:
+            if target not in excluded_list:
+                included_files.append(targets[0])
+    return(included_files)
```

Comment:
Can you change this to `return included_files`, just to be consistent w/ the other code 👍